project4/index.html

```diff
@@ -0,0 +1,383 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://use.typekit.net/dtv5spx.css">
+    <title>#womendesigningwhileblack</title>
+</head>
+
+<body id="top">
+
+    <header>
+
+        <a style="font-size:1.75vw;" class="cta-tickets" href="#">tickets</a>
```

Comment:
Curious why you started adding line styles. All the inline style attributes could just be incorporated into your existing `styles.css` 👍